### DISCUSSION TITLE
Is there a guide for using experimentalFragmentVariables ??

### DISCUSSION THREAD

**Discussion by Yummy-sk** [dotansimha/graphql-code-generator#10124] [Q&A]
Hi, thank you for your great work!

I'm wondering if there's any guide on using experimentalFragmentVariables? When I try to use it like this, it keeps throwing compilation errors:

```gql
fragment AwesomeFrag($id: ID!) on Query {
  hello(id: $id) {
    ...
  } 
}
query AwesomeQuery($id: ID!) {
  ...AwesomeFrag(id: $id)
}
```

Reference: #5444

**Comment by saihaj** [comment]
you enabled the config option for this? If so and still errors can you please share a [minimal reproduction](https://en.wikipedia.org/wiki/Minimal_working_example). You can either use [StackBlitz](https://stackblitz.com/), [CodeSandbox](https://codesandbox.io/) or a public GitHub repo.
